### PR TITLE
Update/edit is working

### PR DESCRIPTION
**Pull request by Echo5Mike**

### CODE REVIEW

File: app/javascript/components/pages/AuctionEdit.js [hunk 1]
```diff
@@ -17,37 +18,33 @@ class AuctionEdit extends Component {
...
+    updateAuction[e.target.name] = e.target.value
+    this.setState({ updateAuction: updateAuction })
+  }
 
   handleSubmit = () => {
-    this.props.updateAuction(this.state.form, this.props.id);
-    this.setState({ submitted: true });
-  };
+    this.props.updateAuction(this.state.updateAuction, this.props.id)
+    console.log("Handle Auction Sub.JS:", this.state.updateAuction)
```
Comment (elysemontano) [inline app/javascript/components/pages/AuctionEdit.js:35]: Please remove console log

File: app/javascript/components/pages/AuctionEdit.js [hunk 2]
```diff
@@ -17,37 +18,33 @@ class AuctionEdit extends Component {
...
+    this.setState({ submitted: true })
+  }
 
   render() {
+    console.log("auction Edit Render:", this.state)
```
Comment (elysemontano) [inline app/javascript/components/pages/AuctionEdit.js:40]: Remove this one as well

**Review by elysemontano — APPROVED** [review]
Looks good, please remove those 2 console logs before merging though!

**Review by zan-rosenthal — APPROVED** [review]

File: app/javascript/components/pages/AuctionEdit.js (hunk 1 above)
Comment (Echo5Mike) [reply app/javascript/components/pages/AuctionEdit.js:35]: Logs have been removed

**Pull request merged**